ReactiveFeedback/SignalProducer+System.swift

```diff
@@ -29,8 +29,23 @@ extension SignalProducer where Error == NoError {
 
             return SignalProducer<Event, NoError>(Signal.merge(events))
                 .scan(initial, reduce)
-                .prefix(value: initial)
```

Comment:
Are you saying that `prefix` operator starts source producer before you even subscribe it `prefix`?

Reply:
No. LHS is started only after RHS completes, and that's the core of the issue.

Reply:
RHS (the prefixed initial) sends out a value, feedbacks receive such value and produce an event, and the said event may be dequeued by the feedback loop queue before LHS (the reducer) starts on the initialising queue.